### PR TITLE
feat: add viewer page

### PR DESCRIPTION
**Pull request by bhajneet**
### Summary of PR
adds a viewer page

### Tests for unexpected behavior
- iphone, ipad, and desktop views

### Time spent on PR
- about an hour thinking through viewer page
- about an hour researching parallax
- about an hour for the actual code
- 2 hours for viewer translation svg
- About 0.5 hours for viewer update svg

### CODE REVIEW

File: src/pages/viewer.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+const useStyles = createUseStyles( () => ( {
+  ib: {
+    display: 'inline-block',
+  },
+  heroMedia: {
+    borderRadius: 8,
+  },
+  hint: {
+    color: '#CCCCCC',
```
Comment (saihaj) [inline src/pages/viewer.tsx:23]: should we put this in colors constants file?

**Review by saihaj — CHANGES_REQUESTED** [review]

File: src/pages/viewer.tsx (hunk 1 above)
Comment (bhajneet) [reply src/pages/viewer.tsx:23]: I need to clean up colors a bit, still not happy with current components for layout, so need to redo those as well.

Comment (saihaj) [reply src/pages/viewer.tsx:23]: then we can just refactor this later

**Comment by bhajneet** [comment]
@saihaj everything but the sub navbar is complete. Can do that in a new PR, this page is ready to go for production imo.

**Review by saihaj — APPROVED** [review]
great work 🔥

**Pull request merged**